calico/datadog_checks/calico/metrics.py

```diff
@@ -0,0 +1,20 @@
+METRIC_MAP = {
+    "felix_active_local_endpoints": "felix_active_local_endpoints",
+    "felix_active_local_policies": "felix_active_local_policies",
+    "felix_active_local_selectors": "felix_active_local_selectors",
+    "felix_active_local_tags": "felix_active_local_tags",
```

Comment:
```suggestion
    "felix_active_local_endpoints": "felix.active.local_endpoints",
    "felix_active_local_policies": "felix.active.local_policies",
    "felix_active_local_selectors": "felix.active.local_selectors",
    "felix_active_local_tags": "felix.active.local_tags",
```
https://docs.datadoghq.com/developers/guide/what-best-practices-are-recommended-for-naming-metrics-and-tags/#rules-and-best-practices-for-naming-metrics